### PR TITLE
V0.3 & v0.4 support

### PR DESCRIPTION
**Pull request by will-moore**
Re-opening old PR...

Now supports v0.3 and v0.4 and included redirect logic from #6 (now closed).

To test (e.g. on merge-ci server):

 - Image -> open with... vizarr or NGFF validator
 - In both cases you should get redirected so that the `source` is an absolute URL (not relative):
 - BUT: there is an issue on merge-ci where the absolute URL is `source=http` not `source=https`

E.g. 
https://merge-ci.openmicroscopy.org/web/zarr/vizarr/?source=http://merge-ci.openmicroscopy.org/web/zarr/v0.4/image/211615.zarr

For the apps to work, you need to edit the `http` to `https`. This shouldn't be an issue on production servers.

 - Then, images should view correctly in vizarr (except for Big images where levels are not 2x scaled - e.g. svs)
 - And images should be valid in the validator
 - Editing the `v0.4` -> `v0.3` should also work with both viewers]
 - Also updated README

### CODE REVIEW

**Comment by joshmoore** [comment]
> For the apps to work, you need to edit the `http` to `https`. This shouldn't be an issue on production servers.

I don't follow, @will-moore. What URL does one use to validate via omero-web-zarr?

File: omero_web_zarr/static/omero_web_zarr/openwith.js [hunk 1]
```diff
@@ -2,5 +2,5 @@
 // Here we configure a url provider. This function will be passed the selected
 // objects and the base url that was specified in the 'Open with' configuration above.
 OME.setOpenWithUrlProvider("web_zarr_vizarr", function (selected, url) {
-    return `${url}vizarr/?source=${url}image/${selected[0].id}.zarr`
```
Comment (joshmoore) [inline omero_web_zarr/static/omero_web_zarr/openwith.js:5]: Does the previous unversioned URL still work?

**Review by joshmoore — COMMENTED** [review]
Looks to be one minor issue with the (new?) validator.

Otherwise, :+1: for having support of the newer versions for testing.

Probably the next step is to consider where we want this to go, especially when mixed with, e.g., storing thumbnails and pyramids in Zarr internally since there would be existing data we could make use of.

File: omero_web_zarr/static/omero_web_zarr/openwith.js (hunk 1 above)
Comment (will-moore) [reply omero_web_zarr/static/omero_web_zarr/openwith.js:5]: No, only v0.3 and v0.4 supported now.

**Comment by will-moore** [comment]
Yes, great idea to simply serve the existing files if the original data is OME-NGFF.

**Pull request merged**